project/Settings.scala

```diff
@@ -78,13 +82,14 @@ object Settings {
   val googleSettings = commonSettings ++ List(
     name := "workbench-google",
     libraryDependencies ++= googleDependencies
-  ) ++ versionSettings ++ noPublishSettings
+  ) ++ versionSettings ++ noPublishSettings ++ java11BuildSettings
 
   //the full list of settings for the rawlsModel project (see build.sbt)
   //coreDefaultSettings (inside commonSettings) sets the project name, which we want to override, so ordering is important.
   //thus commonSettings needs to be added first.
   val modelSettings = cross212and213 ++ commonSettings ++ List(
     name := "rawls-model",
+    javacOptions ++= Seq("--release", "8"), // has to publish a java 8 artifact
```

Comment:
did you figure out who is consuming this? Does it actually need to be java 8?

Reply:
There are [a few consumers](https://github.com/search?p=1&q=%22rawls-model%22&type=Code), but the main potential problem would be workbench-libs, if I understand correctly